packages/web/src/components/ui/Hero/Hero.tsx

```diff
@@ -63,9 +65,11 @@ export default function Hero({ suggestions, onSubmit = () => {}, loading = false
           {suggestions && (
             <div className={styles.suggestions}>
               {suggestions.map((suggestion) => (
-                <Chip key={suggestion} variant='suggest' size='medium'>
-                  {suggestion}
-                </Chip>
+                <Link to={`/scan/${suggestion}`}>
```

Comment:
This is indeed a desired behavior, however, trying to push through `onSumbit` seems more reasonable, i suggest leaving a TODO here

Reply:
I'd like to have an instant redirect on click. What do you want to have in TODO?

Reply:
Nvm, I thought this was the SearchBar component itself, not much of a problem in this case. Just wanted to highlight that we have a `onSubmit` along with another thing that somewhat mimics `onSubmit` inside the component in form of these links, seems weird.